### DISCUSSION TITLE
Snap Edges & Corners Happening in Middle of Screen

### DISCUSSION THREAD

**Discussion by oddeonetwerx** [rxhanson/RectanglePro-Community#476] [Q&A]
I just configured Snap Areas for the first time. I have one at the top of my screen and one in the bottom right corner. However, as soon as I start dragging a window, even in the middle of the scream, my windows start snapping. This prevents me from doing any window organization because no matter where I'm at it snaps to one of those two. I thought the expected behavior would be that I have to actually drag the window to the top or the bottom right corner for it to snap. Any guidance on what might be causing this issue and how to resolve it? 

I'm on the newest version of Rectangle Pro and I'm also on the newest version of macOS Sonoma.

Thank you, 
Neil
<img width="630" alt="Screenshot 2024-03-24 at 11 09 22 AM" src="https://github.com/rxhanson/RectanglePro-Community/assets/48160946/815c3a79-3f1a-40b1-b28e-f4b7efa57839">

**Comment by rxhanson** [comment]
It sounds like you have checked the box for "Use as a snap target" in a custom size & position or app layout that you have created.

You can uncheck the box for those if all you want is to configure them for drag to screen edge snapping.

That checkbox is for the "Snap Targets" section below "Edges & Corners" here. Snap Targets are where you can drag a window to a region, rather than a screen edge.

If this is not your issue, you can send export your config and attach it here, and I can figure out what's going on.

**Reply by oddeonetwerx** [reply]
@rxhanson  That was it. I obviously misunderstood what that button did. Thank you very much for the quick help!
